x-pack/plugins/remote_clusters/public/sections/remote_cluster_form/remote_cluster_form.js

```diff
@@ -0,0 +1,617 @@
+        defaultMessage: `Duplicate seed nodes aren't allowed.`,
+      }));
+    }
+
+    this.setState({
+      localSeedErrors: errors,
+    });
+  };
+
+  onSeedsChange = (seeds) => {
+    this.onFieldsChange({ seeds: seeds.map(({ label }) => label) });
+  };
+
+  onSkipUnavailableChange = (e) => {
+    const skipUnavailable = e.target.checked;
+    this.onFieldsChange({ skipUnavailable });
+  };
+
+  resetToDefault = (fieldName) => {
+    this.onFieldsChange({
+      [fieldName]: defaultFields[fieldName],
+    });
+  };
+
+  renderSeeds() {
+    const {
+      areErrorsVisible,
+      fields: {
+        seeds,
+      },
+      fieldsErrors: {
+        seeds: errorsSeeds,
+      },
+      localSeedErrors,
+    } = this.state;
+
+    const { intl } = this.props;
+
+    // Show errors if there is a general form error or local errors.
+    const areFormErrorsVisible = Boolean(areErrorsVisible && errorsSeeds);
+    const showErrors = areFormErrorsVisible || localSeedErrors.length !== 0;
+    const errors = areFormErrorsVisible ? localSeedErrors.concat(errorsSeeds) : localSeedErrors;
+
+    const formattedSeeds = seeds.map(seed => ({ label: seed }));
+
+    return (
+      <EuiDescribedFormGroup
+        title={(
+          <EuiTitle size="s">
+            <h4>
+              <FormattedMessage
+                id="xpack.remoteClusters.remoteClusterForm.sectionSeedsTitle"
+                defaultMessage="Seed nodes"
+              />
+            </h4>
+          </EuiTitle>
+        )}
+        description={(
+          <Fragment>
+            <p>
+              <FormattedMessage
+                id="xpack.remoteClusters.remoteClusterForm.sectionSeedsDescription1"
+                defaultMessage="When connecting to this remote cluster, its cluster state will be
+                retrieved from one of its seed nodes so that by default up to three gateway nodes
+                are selected to be connected to as part of remote cluster requests."
+              />
+            </p>
+          </Fragment>
+        )}
+        fullWidth
+      >
+        <EuiFormRow
+          label={(
+            <FormattedMessage
+              id="xpack.remoteClusters.remoteClusterForm.fieldSeedsLabel"
+              defaultMessage="IP addresses or hostnames"
+            />
+          )}
+          helpText={(
+            <FormattedMessage
+              id="xpack.remoteClusters.remoteClusterForm.sectionSeedsHelpText"
+              defaultMessage="Seed nodes can be defined as IP addresses or host names, but they
+                must contain a port, e.g. 127.0.0.1:9400 or localhost:9400."
+            />
+          )}
+          isInvalid={showErrors}
+          error={errors}
+          fullWidth
+        >
+          <EuiComboBox
+            noSuggestions
+            placeholder={intl.formatMessage({
+              id: 'xpack.remoteClusters.remoteClusterForm.fieldSeedsPlaceholder',
+              defaultMessage: '127.0.0.1:9400',
```

Comment:
__question:__ is there any reason why this should be a localizable string? Looks like a fixed non-localizable placeholder.